test/smoke_test.rb

```diff
@@ -33,6 +35,25 @@ def test_it_finds_all_movies
     assert_equal Movie.all.size, search.result.size
   end
 
+  def test_scope_filter
+    search = MovieSearch.new({'search' => { 'classics' => 'classics'}})
+    assert_equal 6, search.result.size
+    search = MovieSearch.new({'search' => { 'classics' => 'non_classics'}})
+    assert_equal 34, search.result.size
+    search = MovieSearch.new({'search' => { 'classics' => ['non_classics', 'classics']}})
+    assert_equal 0, search.result.size
```

Comment:
Idealerweise gäbe es halt ein flag was sagt and oder or ich weiß aber auch spontan nicht wie ich in rails scopes verOR